scripts/elasticsearch_nodes.py

```diff
@@ -0,0 +1,127 @@
+"""
+This script is referenced by TerraForm as an external data source. It prints the
+current deployment's Elasticsearch node IDs as JSON to STDOUT. If the list of
+node IDs is less than the number of placeholders in the Cloudwatch dashboard
+template, the output is padded with 'blank' entries.
+Example output:
+{"nodes": "tVsGH2BSXXYe9sMLEYRcIv,ydG9meZMAGkTnAlPgLljz8,blank,blank"}
```

Comment:
Shouldn't this be

```suggestion
{"nodes": ["tVsGH2BSXXYe9sMLEYRcIv", "ydG9meZMAGkTnAlPgLljz8", "", "blank"]}
```

Reply:
No, Terra requires us to provide a dictionary with only strings for both keys and values.

https://registry.terraform.io/providers/hashicorp/external/latest/docs/data-sources/data_source

> The program must then produce a valid JSON object on stdout, which will be used to populate the result attribute exported to the rest of the Terraform configuration. This JSON object must again have all of its values as strings. On successful completion it must exit with status zero.